core-web/libs/dotcms-scss/shared/_common.scss

```diff
@@ -9,10 +9,14 @@ $field-border-radius: $border-radius-md;
 // TODO: convert to rem but breaks math calcs around
 $field-height: 40px;
 
+$text-area-min-height: 64px;
```

Comment:
Are we using this in more than one place? also rem.

Reply:
no, this is just for the min height of the text area, so it's not being used anywhere else.